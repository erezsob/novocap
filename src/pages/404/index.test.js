import React from 'react';
import { MemoryRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NotFoundPage from './';

describe('<NotFoundPage />', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(
        <Router>
          <NotFoundPage />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
