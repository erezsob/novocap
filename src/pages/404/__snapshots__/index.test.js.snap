// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> renders properly 1`] = `
<div
  className="container"
>
  <div
    className="textBox"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        src="test-file-stub"
      />
    </a>
    <h1
      className="bigText"
    >
      404
    </h1>
    <h3>
      Oooops we didn't find the page you are looking for!
    </h3>
  </div>
</div>
`;
